sparkbox/apprentices-page-next
# feat: Update 16.0 apprentices

## Description

This PR adds missing resources for #41.

## To Test

1. Make sure all PR Checks have passed (Github Actions, Netlify etc).
2. Pull down all related branches.
3. Confirm all tests pass: `npm run test:ci`.
4. Visit the PR site.

- [ ] Confirm all current apprentice photos are present.
